conans/client/build/meson.py

```diff
@@ -201,4 +202,4 @@ def get_version():
             version_str = version_line.rsplit(' ', 1)[-1]
             return Version(version_str)
         except Exception as e:
-            raise ConanException("Error retrieving Meson version: '{}'".format(e))
\ No newline at end of file
+            raise ConanException("Error retrieving Meson version: '{}'".format(e))
```

Comment:
PEP8 doesn't like this change 😭

Reply:
why? what/where you see it?

Reply:
There should be an empty new line at the end of the file. At least that is what pycharm suggests... 😀

Reply:
Here I'm adding the newline, before it wasn't there

Reply:
Really? Always understood it as the other way around 🤣 Sorry then